plugins/org.eclipse.n4js.json.ui/src/org/eclipse/n4js/json/ui/editor/hyperlinking/HyperlinkHelperProvider.java

```diff
@@ -0,0 +1,44 @@
+package org.eclipse.n4js.json.ui.editor.hyperlinking;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.eclipse.jface.text.hyperlink.IHyperlink;
+import org.eclipse.n4js.json.ui.extension.JSONUiExtensionRegistry;
+import org.eclipse.xtext.resource.XtextResource;
+import org.eclipse.xtext.ui.editor.hyperlinking.HyperlinkHelper;
+
+import com.google.inject.Inject;
+
+public class HyperlinkHelperProvider extends HyperlinkHelper {
```

Comment:
We could use the name `JSONHyperlink...` to clearly state this is only applied to JSON file